### DISCUSSION TITLE
Cancel draft workflow from dashboard

### DISCUSSION THREAD

**Discussion by SamueleFurnari** [elsa-workflows/elsa-core#1607] [Q&A]
Hi guys,
I use very often (expecially in presence of smart users), the dashboard to design, manage and monito the workflows. 

I cant find a way to cancel a draft definition without using custom code or without unpublish/publish the workflow. Is there a more convenient way to do this?

Thanks very much!
Samuele

**Comment by sfmskywalker** [comment]
Do you mean that you need the ability to discard a workflow definition draft? 

For example, you create a new workflow that is then saved as a draft automatically as soon as you make some change. Now you wish to abandon this workflow via some discard button.

I think this would be useful to have. If this is what you have in mind as well, then perhaps you could create an issue requesting this feature.
